docker/nginx-gateway.conf.template

```diff
@@ -50,6 +50,10 @@ server {
     # Ensure only https is used for this server
     add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
 
+    # Govern what content can be loaded by the server and from where
+    # Click jacking prevention to be used in addition to X-Frame-Options
+    add_header Content-Security-Policy "default-src 'self'; frame-ancestors 'self'; form-action 'self'; ";
```

Comment:
CSP header is really tricky part for Hawtio. I am sure this policy breaks Hawtio in several ways. Have you tested this fix with a running hawtio-online instance?

Also check what ContentSecurityPolicyFilter in Hawtio standalone does.

Reply:
> CSP header is really tricky part for Hawtio. I am sure this policy breaks Hawtio in several ways. Have you tested this fix with a running hawtio-online instance?
> 
> Also check what ContentSecurityPolicyFilter in Hawtio standalone does.

I have tested this fix. This particular entry is at the top server level hence it is overridden by the entry in the location /online section below.

This one is entered in the /online location, allowing for inline-styles, svg images and inline jpg images:

```
add_header Content-Security-Policy "default-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; frame-ancestors 'self'; form-action 'self'; 
```

Reply:
This is the analysis of the Hawtio [ContentSecurityPolicyFilter](https://github.com/hawtio/hawtio/blob/4.x/hawtio-system/src/main/java/io/hawt/web/filters/ContentSecurityPolicyFilter.java):

Only thing missing was a `data:` entry for font-src - now included.

```
default-src 'self'

// Keycloak not applicable
// OIDC not applicable
// not required as defaults to default
script-src  'self'

style-src 'self' 'unsafe-inline'

font-src 'self' data:

img-src 'self' data:

// Keycloak not applicable
// OIDC not applicable
// not required as defaults to default
connect-src  'self'

// Keycloak not applicable
// OIDC not applicable
// not required as defaults to default
frame-src 'self'

// not required as defaults to default
manifest-src 'self'

// not required as defaults to default
media-src 'self'

// not required as defaults to default
object-src 'self'

// not required as defaults to default
worker-src 'self'

// X-Frame-Options is SAMEORIGIN
frame-ancestors 'self'
```

Reply:
OK, looks good. Let's merge it then.